src/components/Community/Recommendations.tsx

```
@@ -0,0 +1,82 @@
+      <RecommendationsHeader>
+        <img src="/images/recCommsArt.png" />
+        <NorText>Top Communities</NorText>
+      </RecommendationsHeader>
+      <RecommendationsList>
+        {communities.map((com, i) => {
+          const isJoined = !!communityStateValue.mySnippets.find(
+            (item) => item.communityId === com?.id
+          );
+
+          return (
+            <RecommendationsItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)